tests/unittest_protocols.py

```diff
@@ -0,0 +1,24 @@
+from typing_extensions import Protocol
+from screenpy.protocols import Answerable, Forgettable, Performable, ErrorKeeper
+from screenpy_selenium.protocols import Chainable
+
+class Describable(Protocol):
+    def describe(self) -> str:
+        ...
```

Comment:
We won't have access to this Protocol here in `screenpy_selenium` until the newer version of ScreenPy is released. Should we add a comment here to remind ourselves to remove this one once we have the other available?